e2e/cypress/tests/support/api/user.d.ts

```diff
@@ -205,7 +205,7 @@ declare namespace Cypress {
          * @example
          *   cy.apiCreateUser(options);
          */
-        apiCreateUser(options: Record<string, any>): Chainable<{user: UserProfile}>;
+        apiCreateUser(options?: Record<string, any>): Chainable<UserProfile>;
```

Comment:
Moving to `Chainable<UserProfile>` will probably break the other tests.

This was changed in PR #10910 - a.k.a the Support Migration PR

Reply:
Right. It should match the actual implementation. Please update to latest master.